docker/sbtc/signer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM sbtc-build:latest AS build
+FROM debian:bookworm-slim AS runtime
+COPY --from=build /code/sbtc/target/debug/sbtc-signer /usr/local/bin/sbtc-signer
```

Comment:
This doesn't build for me:

```txt
 > [sbtc-signer-2 runtime 2/2] COPY --from=build /code/sbtc/target/debug/sbtc-signer /usr/local/bin/sbtc-signer:
```

I did a bit of spelunking and the image `sbtc-build:latest` contains a `signer`, not `sbtc-signer` binary:

```bash
root@71b0615ffe24:/code/sbtc/target/debug# ls -la | grep signer
-rw-r--r-- 1 root root     6593 Oct 23 08:36 libsigner.d
-rw-r--r-- 2 root root 40059974 Oct 23 08:36 libsigner.rlib
-rwxr-xr-x 2 root root 88620392 Oct 23 08:36 signer
-rw-r--r-- 1 root root     6615 Oct 23 08:36 signer.d
```

Is it a typo or my system is broken in some way?

Reply:
AH, this is because the original was taking from the `demo-cli` branch which had an [`sbtc-signer`](https://github.com/stacks-network/sbtc/blob/feat/demo-cli/signer/Cargo.toml#L8) binary, but when I changed the original target used to cache dependencies to main it no longer has this binary, and when I retested it I must have included caching. So in this it was just using the `sbtc-signer` binary but should have been using the `signer` binary. Shoot!